app/assets/stylesheets/components/forms.scss

```diff
@@ -183,6 +219,18 @@
   }
 }
 
+///////////////////////////////////////////////////
+.crayons-textfield {
```

Comment:
Is a text field an input field or a text area?

Reply:
Correct, you can use `crayons-textfield` for both: `<input type="text">` and `<textarea>`